jhub_apps/templates/404.html

```diff
@@ -0,0 +1,10 @@
+{% extends "page.html" %} {% block title %} 404 Page Not Found {% endblock %} {%
+block main %}
+
+<div class="center-content-text v-center">
+  <h1 class="m-b-0">404 : PAGE NOT FOUND</h1>
+  <h2>Sorry, the page you are looking for does not exist.</h2>
+  <a href="/" role="link">Back to Home</a>
```

Comment:
We should probably use the JupyterHub base_url here instead of `/`. Suggest changing to this:
`href="{{base_url}}"`

Reply:
Done